.github/workflows/quality.yml

```diff
@@ -94,7 +96,20 @@ jobs:
         with:
           cache-encryption-key: ${{ secrets.GRADLE_CACHE_ENCRYPTION_KEY }}
       - name: Run Dependency Analysis
+        id: analyse_dependencies
+        continue-on-error: true
         run: ./gradlew buildHealth
+      - name: Comment Analysis Report
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          path: build/reports/dependency-analysis/build-health-report.txt
+          delete: ${{ steps.analyse_dependencies.outcome == 'success' }}
+      - name: Dependency Analysis failed
+        if: ${{ steps.analyse_dependencies.outcome == 'failure' }}
+        run: |
+          echo "::error ::Dependency Analysis failed."
+          exit 1
```

Comment:
This comes from Media Maestro:
https://github.com/SRGSSR/MediaMaestro/blob/c496a11e3bdd0a4c3207be0f08eafd5d99c585c3/.github/workflows/quality.yml#L202-L212